### PR TITLE
Improved Russian translation

### PR DESCRIPTION
**Pull request by Jokero**
Compared to https://github.com/apidoc/apidoc/pull/237

### CODE REVIEW

File: template/index.html [hunk 1]
```diff
@@ -470,7 +470,7 @@
         <table class="ins">
         <thead>
           <tr>
-            <th style="width: 30%">{{#if _col1}}{{__ _col1}}{{else}}{{__ "Field"}}{{/if}}</th>
+            <th style="width: 30%">{{#if _col1}}{{__ _col1}}{{else}}{{__ "Name"}}{{/if}}</th>
```
Comment (zxqfox) [inline template/index.html]: Feels like it will broke all other translations.

Comment (Jokero) [inline template/index.html]: Ok

**Comment by Jokero** [comment]
All is good

**Comment by rottmann** [comment]
Ok thanks, update it on the next patch.

**Comment by zxqfox** [comment]
@rottmann @Jokero :+1: Thanks!

**Pull request closed without merging**